### PR TITLE
[4.2 re-do] Fix for Broken html in Date of Birth field in User Profile Form

### PR DESCRIPTION
**Pull request by AndySDH**
As requested, here's the Joomla 4.2 Re-Do of Pull Request #28884 (which was for Issue #28883).

### Summary of Changes

This fixes the Date of Birth field in User Profile Edit Form that generates a broken html.

The issue is to due the extra info about the date format being added with broken html around it.

I've also update the language string to be shorter in order to fit the new positioning better. See the original PR #28884 for more details/discussion. 

### Previous result

The previous "hacky" HTML creates an extra control-group, and the actual "controls" is outside of it, so two problems for the price of one.

The goal was to put the info language string on a separate line but this breaks the HTML structure of the form.

### New result

Now the info sits near the label and doesn't break the HTML of the control-group. It does occupy a bit more space (with the default layout) but now the HTML is valid

### Testing Instructions

Simply see the HTML of the edit form of a user profile page with a DOB field enabled.

### Documentation Changes Required

None

### CODE REVIEW

File: administrator/language/en-GB/plg_user_profile.ini [hunk 1]
```diff
@@ -12,7 +12,7 @@ PLG_USER_PROFILE_FIELD_ADDRESS1_LABEL="Address 1"
 PLG_USER_PROFILE_FIELD_ADDRESS2_LABEL="Address 2"
 PLG_USER_PROFILE_FIELD_CITY_LABEL="City"
 PLG_USER_PROFILE_FIELD_COUNTRY_LABEL="Country"
-PLG_USER_PROFILE_FIELD_DOB_LABEL="Date of Birth"
+PLG_USER_PROFILE_FIELD_DOB_LABEL="Birth Date"
```
Comment (brianteeman) [inline administrator/language/en-GB/plg_user_profile.ini]: This should be reverted - it is always Date of Birth unless its a bad translation

Comment (AndySDH) [reply administrator/language/en-GB/plg_user_profile.ini]: Ok, sorry, will do.

**Comment by brianteeman** [comment]
What am I missing? Surely this just needs a regular field of type calendar and no extra fields or layouts

image

image

or could also include a hint

image


image

**Comment by AndySDH** [comment]
To be honest I agree. But I wasn't / am not sure how to go about completely removing a layout. So if you know how to do so, please be my guest, I do agree with you that a simple hint would be enough.

**Comment by Kostelano** [comment]
Don't add `hint="YYYY-MM-DD"` because `DD.MM.YYYY` is correct for Russian/Ukrainian and many other languages.

Screenshot_1

**Comment by brianteeman** [comment]
The hint is a translatable string

**Comment by AndySDH** [comment]
@brianteeman Can you make a new PR with your idea of making it a calendar field with the hint?

**Comment by brianteeman** [comment]
alternative created as requested #38065

**Comment by AndySDH** [comment]
I'll close this in favor of @brianteeman's PR.

**Pull request closed without merging**